interpreter/php/decode_amd64.c

```diff
@@ -16,16 +16,17 @@ int retrieveJITBufferPtr(const uint8_t * const code, const size_t codesize,
                          const uint64_t rip_base, uint64_t * const buffer_ptr,
                          uint64_t * const size_ptr) {
   ZydisDecoder decoder;
-  ZydisDecoderInit(&decoder, ZYDIS_MACHINE_MODE_LONG_64, ZYDIS_ADDRESS_WIDTH_64);
+  ZydisDecoderInit(&decoder, ZYDIS_MACHINE_MODE_LONG_64, ZYDIS_STACK_WIDTH_64);
   ZydisDecodedInstruction instr;
+  ZydisDecodedOperand operands[ZYDIS_MAX_OPERAND_COUNT];
   ZyanUSize instruction_offset = 0;
 
   // These are to check that we've written to both pointers.
   int written_to_buffer_ptr = 0;
   int written_to_size_ptr = 0;
   
-  while(ZYAN_SUCCESS(ZydisDecoderDecodeBuffer(&decoder, code + instruction_offset,
-                                              codesize - instruction_offset, &instr))) {
+  while(ZYAN_SUCCESS(ZydisDecoderDecodeFull(&decoder, code + instruction_offset,
+                                            codesize - instruction_offset, &instr, operands))) {
```

Comment:
The biggest difference between Zydis v3 and v4 API is that operand decoding is now optional and the operand array no longer lives in the instruction struct.